### ISSUE TITLE
DL3013 does not allow  --ignore-installed argument for pip install

### ISSUE THREAD

**Issue by ferrarimarco** [hadolint/hadolint#157]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

`RUN pip install --ignore-installed pip==9.0.1` not to trigger any error.

### Actual behavior

I get a false positive on `DL3013` when the [`-I (--ignore-installed)`](https://pip.pypa.io/en/stable/reference/pip_install/#cmdoption-i) option is specified.

### Steps to reproduce the behavior

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:** `Haskell Dockerfile Linter v1.2.6-0-ge03cbb0`

```bash
Dockerfile:56 DL3013 Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`
```

### References

#153 #115
